styles.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+}
```

Comment:
- [ ] I think you need to add a bit of padding to the container so that your work will look like the given design. 😄